sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -71,6 +71,9 @@
 from tests.integration.feature_repos.universal.online_store.bigtable import (
     BigtableOnlineStoreCreator,
 )
+from tests.integration.feature_repos.universal.online_store.cassandra import (
```

Comment:
Revert the changes for now. We may need to follow the documentation in docs/how-to-guides/customizing-feast/adding-support-for-a-new-online-store.md to support integration tests.